### ISSUE TITLE
`webpack` configuration and `devDependencies` usage

### ISSUE THREAD

**Issue by joshua-dean** [SenteraLLC/ulabel#164]
See discussion [here](https://github.com/SenteraLLC/ulabel/pull/161#discussion_r1765591869) and [here](https://github.com/SenteraLLC/ulabel/pull/161#issuecomment-2361161895). 

Currently "minimizing"/"uglifying" logic in `webpack.config.js` is commented out, so the build process should probably revisited and documented (maybe `webpack` isn't even needed?). a

Conversation stems from `package.json` missing `typescript` as a dependency, but as ULabel (currently) gets packed with `webpack`, technically _everything_ should be in `devDependencies` as it all gets packed when pushed to `npm`. [This stackoverflow answer](https://stackoverflow.com/a/55133339) indicates there's some nice categories when using `npm` itself, so maybe that is worth looking into.

tl;dr Laziest solution is chuck everything in `devDependencies` and clean up `webpack.config.js`.
